docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: "2"
+services:
+  description:
+    environment:
+      - MONGO_SERVER=172.18.0.2
+      - MONGO_PORT=27017
+    container_name: description
+    restart: always
+    build: .
+    ports:
+      - "3002:3002"
+    links:
+      - mongo
+  mongo:
+    container_name: mongo
+    image: mongo:3.4.13
+    ports:
+      - "27027:27017"
```

Comment:
must you expose the db to the host system?

Reply:
It is useful for testing purposes. Allows for accessing the database using mongo client.